.github/bucket.yaml

```diff
@@ -0,0 +1,63 @@
+AWSTemplateFormatVersion: '2010-09-09'
+
+Description: |
+  Maintainers Only: Create a S3 bucket used for hosting the artifacts in the AWS SAM package process
+
+
+Resources:
+  ArtifactBucket:
+    Type: AWS::S3::Bucket
+    Properties:
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+          - ServerSideEncryptionByDefault:
```

Comment:
Should we be using [Amazon S3 Bucket Keys](https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingKMSEncryption.html#:~:text=When%20you%20configure%20server%2Dside%20encryption%20using%20AWS%20KMS%20(SSE%2DKMS)%2C%20you%20can%20configure%20your%20buckets%20to%20use%20S3%20Bucket%20Keys%20for%20SSE%2DKMS.%20Using%20a%20bucket%2Dlevel%20key%20for%20SSE%2DKMS%20can%20reduce%20your%20AWS%20KMS%20request%20costs%20by%20up%20to%2099%20percent%20by%20decreasing%20the%20request%20traffic%20from%20Amazon%20S3%20to%20AWS%20KMS.) to save on some costs? Is the way to go?

Reply:
AWS S3 bucket keys are relevant only when using KMS encryption